lib/Executor/Benchmark/template/remote.template

```diff
@@ -25,21 +28,21 @@ $benchmark = new $class();
 
 // run before methods
 foreach ($beforeMethods as $beforeMethod) {
-    $benchmark->$beforeMethod({{ parameters }});
+    $benchmark->$beforeMethod($parameters);
 }
 
 // run warmup if required
 if ($warmup) {
     for ($i = 0; $i < $warmup; $i++) {
-        $benchmark->{{ subject }}({{ parameters }});
+        $benchmark->{{ subject }}();
     }
 }
 
 // run benchmark
 $startTime = microtime(true);
 
 for ($i = 0; $i < {{ revolutions }}; $i++) {
-    $benchmark->{{ subject }}({{ parameters }});
+    $benchmark->{{ subject }}($parameters);
```

Comment:
passing $parameters probably incurs an overhead, have two loops one with and one without